source/_components/light.avion.markdown

```diff
@@ -21,6 +21,10 @@ To enable these lights, add the following lines to your `configuration.yaml` fil
 # Example configuration.yaml entry
 light:
   - platform: avion
+    username: testuser@fakedomain.com
```

Comment:
Basic example should only contain *Required* variables or the minimal example to get going. Feel free to also add an advanced example if so required.